### PR TITLE
[TACHYON-662] Clean up Worker General Servlet

### PR DESCRIPTION
**Pull request by danielfree**
Add an UiWorkerInfo class to store the attributes, this can clean up the jsp.

### CODE REVIEW

**Comment by gpang** [comment]
ok to test

**Comment by calvinjia** [comment]
+1 LGTM, thanks for the patch!

File: servers/src/main/java/tachyon/web/WebInterfaceWorkerGeneralServlet.java [hunk 1]
```diff
@@ -66,16 +66,39 @@ public long getUsedBytes() {
     }
   }
 
+  public static class UiWorkerInfo {
+    public static final boolean DEBUG = Constants.DEBUG;
+    private final String mWorkerAddress;
+    private final long mStartTimeMs;
+    public static final String VERSION = Version.VERSION;
```
Comment (gpang) [inline servers/src/main/java/tachyon/web/WebInterfaceWorkerGeneralServlet.java]: Can you move this up so it is grouped with the other "static final"?

File: servers/src/main/java/tachyon/web/WebInterfaceWorkerGeneralServlet.java [hunk 2]
```diff
@@ -66,16 +66,39 @@ public long getUsedBytes() {
...
-  private final transient InetSocketAddress mWorkerAddress;
-  private final transient long mStartTimeMs;
+  private final UiWorkerInfo mUiWorkerInfo;
 
   public WebInterfaceWorkerGeneralServlet(BlockDataManager blockDataManager,
       InetSocketAddress workerAddress, long startTimeMs) {
     mBlockDataManager = blockDataManager;
-    mWorkerAddress = workerAddress;
-    mStartTimeMs = startTimeMs;
+    mUiWorkerInfo = new UiWorkerInfo(workerAddress.toString(),startTimeMs);
```
Comment (gpang) [inline servers/src/main/java/tachyon/web/WebInterfaceWorkerGeneralServlet.java]: Could you add a space after ","?

File: servers/src/main/java/tachyon/web/WebInterfaceWorkerGeneralServlet.java [hunk 3]
```diff
@@ -92,16 +115,7 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
...
-
-    request.setAttribute("workerAddress", mWorkerAddress.toString());
-
-    request.setAttribute("uptime",
-        Utils.convertMsToClockTime(System.currentTimeMillis() - mStartTimeMs));
-
-    request.setAttribute("startTime", Utils.convertMsToDate(mStartTimeMs));
-
-    request.setAttribute("version", Version.VERSION);
+    request.setAttribute("workerInfo",mUiWorkerInfo);
```
Comment (gpang) [inline servers/src/main/java/tachyon/web/WebInterfaceWorkerGeneralServlet.java]: Could you add a space after ","?

File: servers/src/main/webapp/worker/general.jsp [hunk 4]
```diff
@@ -16,6 +17,7 @@
 <script src="../js/bootstrap.min.js"></script>
 <div class="container-fluid">
   <% request.setAttribute("useWorkerHeader", "1"); %>
+  <% UiWorkerInfo workerInfo = (UiWorkerInfo)request.getAttribute("workerInfo"); %>
```
Comment (gpang) [inline servers/src/main/webapp/worker/general.jsp]: Could you add a space after the cast, "(UiWorkerInfo)"?

**Comment by gpang** [comment]
Thanks for the patch @danielfree . I just had a few minor style comments.

**Comment by danielfree** [comment]
@gpang Thanks for the comment, I've committed the style fix :)

**Comment by gpang** [comment]
LGTM

**Pull request merged**